src/test/java/IMDbBirthdays.java

```diff
@@ -0,0 +1,133 @@
+import com.codeborne.selenide.Configuration;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.WebDriverRunner;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class IMDbBirthdays {
+
+    @BeforeClass
+    public void setUp() {
+
+        Configuration.browser = "chrome";
+        Configuration.browserSize = "1920x1080";
+    }
+
+    @Test
+    public void fetchCelebritiesWithTodayBirthday() {
+        openIMDbHomepage();
+        openBornTodayMenu();
+        ElementsCollection celebrities = fetchCelebrityElements();
+        printCelebrityInfo(celebrities);
+    }
+
+    private void openIMDbHomepage() {
+        Selenide.open("https://www.imdb.com");
+        System.out.println("IMDb homepage opened.");
+    }
+
+
+    private void openBornTodayMenu() {
+        clickMenuButton();
+        clickBornTodayLink();
+    }
+
+
+    private void clickMenuButton() {
+        try {
+            WebElement menuButton = Selenide.$(By.id("imdbHeader-navDrawerOpen"));
+            menuButton.click();
+            System.out.println("Menu opened.");
+        } catch (Exception e) {
+            System.err.println("Error opening menu: " + e.getMessage());
+        }
+    }
+
+    private void clickBornTodayLink() {
+        try {
+            WebElement bornTodayLink = Selenide.$(By.xpath("//span[text()='Born Today']"));
```

Comment:
I myself not a fan of xpath, but sometimes it might be move convenient, but there is a way to use it properly:
`SelenideElement bornTodayElement = $x("//span[text()='Born Today']");`